website_analytics/security/google_analytics_security.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <odoo>
+    <record id="group_google_analytics_user" model="res.groups">
+        <field name="name">Google Analytics User</field>
+        <field name="category_id" ref="base.module_category_website"/>
+    </record>
+</odoo>
```

Comment:
This is already declared, and we can't put an `<odoo>` in an `<odoo>`

Reply:
In my opinion this file can be delete, we can manage internal users permission using only ir.model.access.csv